rat_bot.py

```diff
@@ -81,7 +81,7 @@ async def on_message(message):
         except Exception:
             await message.channel.send(
                 "Squeek squeek... "
-                f"(Something went wrong, make {admin} fix it...)",
+                f"(Something went wrong, make <@{admin_id}> fix it...)",
```

Comment:
Actually, looking at this I'm not entirely sure what's going on here? Does this even work? I thought the signature of `message.channel.send` was something like `(content: str, other_stuff_here)`...

Reply:
Resolving - I think it just works. I think I looked at this and just thought "hey that kinda looks odd" but [the signature seems to allow this](https://discordpy.readthedocs.io/en/stable/api.html?highlight=channel%20send#discord.TextChannel.send):
> `await send(content=None, *, tts=False, embed=None, file=None, files=None, delete_after=None, nonce=None, allowed_mentions=None, reference=None, mention_author=None)`